### PR TITLE
Permissions in admin templates

### PR DESCRIPTION
**Pull request by reinaris**
I implemented basic permissions in the admin templates.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://reinaris-admin-product-abilities.spree-spree-9ed724b77adec04c49c2.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://reinaris-admin-product-abilities.spree-spree-9ed724b77adec04c49c2.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: core/app/models/spree/ability.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@ def initialize(user)
       alias_action :new_action, to: :create
       alias_action :show, to: :read
       alias_action :index, :read, to: :display
-
+      alias_action :create, :update, :destroy, to: :modify
```
Comment (JDutil) [inline core/app/models/spree/ability.rb]: Why not just use :manage ?

**Comment by JDutil** [comment]
Would like other peoples thoughts on the alias names.  assign might not be explicit enough I'm not sure, but conceptually I'm alright with the change.

File: core/app/models/spree/ability.rb (hunk 1 above)
Comment (reinaris) [inline core/app/models/spree/ability.rb]: @JDutil I can't discover what :manage stands for, is it exactly the same (create, update, destroy)? In that case we should :)

Comment (simmsy) [inline core/app/models/spree/ability.rb]: @JDutil manage implies you can do all actions on an object. For example you may be able to create, edit or destroy an order (therefore view it's edit screen) but you cannot approve an order (which would be included with the manage action). modify implies you can mutates an object BUT not necessarily perform all actions.

**Comment by simmsy** [comment]
@JDutil other more programming friendly term for assign would be relate?

**Comment by peterberkenbosch** [comment]
Personally I think assign is the correct wording here. That's exactly what an user is allowed or disallowed to do right? :+1:

**Comment by reinaris** [comment]
I made this PR more general for all admin templates.

I deleted the new alias from this PR to make it more easy.
Will open a new PR for this, so we can discus there.

**Comment by athal7** [comment]
:+1:

**Comment by peterberkenbosch** [comment]
Like it!, just 2 failing specs left @reinaris. Could you take look?

```
1) Tax Categories admin creating new tax category should be able to create new tax category
     Failure/Error: Unable to find matching line from backtrace
     ActionView::Template::Error:
       undefined local variable or method `tax_category' for #<#<Class:0x0000000fd99e50>:0x0000001177e230>
     # /home/ubuntu/spree/core/app/helpers/spree/base_helper.rb:68:in `method_missing'
     # ./app/views/spree/admin/tax_categories/index.html.erb:40:in `_1c744aa660a9e8aa540acb51a67856b2'

  2) Tax Categories admin creating new tax category should show validation errors if there are any
     Failure/Error: Unable to find matching line from backtrace
     ActionView::Template::Error:
       undefined local variable or method `tax_category' for #<#<Class:0x0000000fd99e50>:0x00000010a6d208>
     # /home/ubuntu/spree/core/app/helpers/spree/base_helper.rb:68:in `method_missing'
     # ./app/views/spree/admin/tax_categories/index.html.erb:40:in `_1c744aa660a9e8aa540acb51a67856b2'
```

**Comment by reinaris** [comment]
@peterberkenbosch I will have a look tomorrow!

**Comment by reinaris** [comment]
@peterberkenbosch locally they just pass. From the backend app I run ` bundle exec rspec spec/features/admin/configuration/tax_categories_spec.rb`.
No errors.

**Comment by reinaris** [comment]
@peterberkenbosch fixed it!

**Comment by reinaris** [comment]
New errors showed up, it this because I rebased against master, because I didn't changed anything else then those tax_category fix.

**Pull request closed without merging**